R/calculateRelativeFC.R

```diff
@@ -118,9 +122,7 @@ calculateRelativeFC <- function(se, design, coef = NULL, contrast = NULL,
         ## WT rows
         tmp0 <- dge$counts[WTrows, , drop = FALSE]
         dge <- edgeR::scaleOffset(dge, log(colSums(tmp0)))
-    } else {
-        stop("Unknown 'normMethod'")
-    }
+    } 
```

Comment:
just a cosmetic thing: similarly to this replacement of the last else clause with an `.assertScalar` before the if, this could also be cone for `method` (if-else clauses starting at line 128)

Reply:
Resolved in a371a43